### DISCUSSION TITLE
Support Fine tuning and Refinement

### DISCUSSION THREAD

**Discussion by briansunter** [briansunter/logseq-plugin-gpt3-openai#164] [Ideas]
- [ ] Train it on your own notes

- [ ] Train on file

https://beta.openai.com/docs/guides/fine-tuning

**Comment by arminta7** [comment]
Is there any update on this?

**Comment by briansunter** [comment]
I have a mini project roadmap right here if you'd like to follow the progress and rough order I'm thinking about working on the features. https://github.com/users/briansunter/projects/1
